Makefile.common

```diff
@@ -201,7 +201,7 @@ common-unused:
 .PHONY: common-build
 common-build: promu
 	@echo ">> building binaries"
-	$(PROMU) build --prefix $(PREFIX) $(PROMU_BINARIES)
+	GO111MODULE=$(GO111MODULE) $(PROMU) build --prefix $(PREFIX) $(PROMU_BINARIES)
```

Comment:
Doesn't this need the `--cgo` flag, too?

Reply:
`/ Build Prometheus for all architectures (11) (pull_request)` these jobs tend to fail when I add the flag